Week1/HTML.md

```diff
@@ -0,0 +1,260 @@
+2. The closing tag - `</p>`: This is the same as the opening tag, except that it includes a forward slash before the element name `</>`. This states where the element ends — in this case where the end of the paragraph is. Failing to include a closing tag is a common beginner error and can lead to strange results.
+3. The content: This is the content of the element, which in this case is just text.
+4. The element: The opening tag plus the closing tag plus the content equals the element.
+
+## Basic structure of an HTML page
+
+![html basic structure](https://user-images.githubusercontent.com/22002193/69439363-72556d00-0d4f-11ea-8779-2005751cdd10.png)
+
+The three tags that you need for your HTML document are `<html>`, `<head>`, and `<body>`.
+
+1. The `<html></html>` tag is the highest level element that encloses every HTML page.
+2. The `<head></head>` tag holds meta information such as the page’s title and charset.
+3. Finally, the `<body></body>` tag encloses **all the content that appears on the page**.
+
+## How does HTML work?
+
+HTML documents end with the .html or .htm extension. You can view it using any web browser. The browser reads the HTML file and renders the content for users to view it.
+
+### Task: Build a basic HTML page
+
+- open your favourite editor and create a file called `my-first-page.html`
+- type `html` in the page and press the `tab` key, this will automatically generate an html page for you.
+
+The result should look like the following:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title></title>
+  </head>
+  <body></body>
+</html>
+```
+
+_Note: We will explore all the tags later_
+
+- In the `<body>` tag add an `<h1>`
+
+Headings are defined with the `<h1>` to `<h6>` tags.
+
+`<h1>` defines the most important heading and `<h6>` defines the least important heading.
+
+Your code should look something like this:
+
+```html
+<body>
+  <h1>The Main Heading</h1>
+</body>
+```
+
+- Right click on your html file and click on `open in browser`, this should open the browser with the result of this file. You should see the h1 heading with `The Main Heading` on the page.
+
+<details>
+  <summary>If you are using Atom</summary>
+  To be able to preview your html file in a browser, install the _open-in-browsers_ package by going to:
+  
+  Atom -> Preferences -> Install
+  </details>
+
+- Let's explore this a bit more, in the browser, open the inspector. You can do so by either pressing `f12` or right clicking on the page and then inspect or pressing `ctrl+shift+i`.
+
+  - You should see be able to see the elements you created before (raw html), in the elements tab. You can edit/delete/add elements here like you did in your editor.
+
+  - Change the elements in the browser from the inspector, delete the current `h1` tag and add a `div` tag.
```

Comment:
They should know that there are a few options and that the inspector is not just a way to see the elements but that you can also manipulate things from. 
It's not that confusing

Reply:
it is very confusing since you almost never going to manually add an html element to the browser, and it is introducing a new development environment. why not just tell them to add the div in VSCode ?

Reply:
@kira-abl, after some debate we agreed to remove the editing from the browser.

- keep the note
- they should add a div from the editor and then continue to the other steps

Reply:
@ So the only change will be that they will add a div in their editor instead of the browser? All the rest stays the same?

Reply:
Yes

Reply:
@shiryz Done.